euth/contrib/templatetags/contrib_tags.py

```diff
@@ -24,3 +25,39 @@ def limited_paginator(page_obj):
         current_index < last_index - ADDITIONAL_PAGES else last_index
 
     return page_obj.paginator.page_range[start_index:end_index]
+
```

Comment:
most probably this is the wrong spot. i would like to extract the data for the steps of the howto screen, but no idea how to make it available properly so i misused `contrib_tags` for now. @Rineee maybe you have a better location or idea to organize this?

Reply:
I would say as the contents won't change we can just put it directly into the carousel in the template, what do you think @rine?

Reply:
I think I would add it to the form as a property as its done here with the formsections https://github.com/liqd/a4-opin/blob/9fe7f4f99bad8814415dde96873b43d4069eaaf5/euth/accounts/forms.py#L24-48
The form is here https://github.com/liqd/a4-opin/blob/9fe7f4f99bad8814415dde96873b43d4069eaaf5/euth/users/forms.py#L9
Then when you include the howto.hmtl, you can pass the howto_items likes this: {% include "howto.html" with howto_items=form.howto_items %}

Reply:
Nice! just saw that after i pushed the last commit. I will have a look on that with the current subtask and maybe change it. but the current version (directly in carousel) works also and is okay to be merged i think.